src/snowflake/snowpark/modin/pandas/series.py

```diff
@@ -781,25 +781,37 @@ def apply(
 
         return self.__constructor__(query_compiler=new_query_compiler)
 
-    @series_not_implemented()
     def argmax(self, axis=None, skipna=True, *args, **kwargs):  # noqa: PR01, RT01, D200
         """
         Return int position of the largest value in the Series.
         """
         # TODO: SNOW-1063347: Modin upgrade - modin.pandas.Series functions
-        result = self.idxmax(axis=axis, skipna=skipna, *args, **kwargs)
-        if np.isnan(result) or result is pandas.NA:
+        if self._query_compiler.has_multiindex():
+            # The index is a MultiIndex, current logic does not support this.
+            ErrorMessage.not_implemented(
+                "Series.argmax is not yet supported when the index is a MultiIndex."
+            )
+        result = self.reset_index(drop=True).idxmax(
```

Comment:
Given that we are calling `reset_index` here, it seems that `result` can never be None or non-int. So I'm wondering if the check below is ever satisfied.

Reply:
result is None or a non-integer when idxmax does not find a matching value, so the check still needs to exist